tests/acceptance/install/01-InstallJoomlaCest.php

```diff
@@ -1,13 +1,22 @@
 <?php
 /**
  * @package     redCORE
- * @subpackage  Cept
- * @copyright   Copyright (C) 2008 - 2016 redCOMPONENT.com. All rights reserved.
+ * @subpackage  Cest
+ * @copyright   Copyright (C) 2008 - 2019 redCOMPONENT.com. All rights reserved.
  * @license     GNU General Public License version 2 or later; see LICENSE.txt
  */
 
+use Step\Acceptance\TemplateSteps as AdminTester;
```

Comment:
Hi @OscarIboy. I see code run good . But I think you can use TemplateSteps direct needn't use 'as' It will perfect

Reply:
@NguyenTrang091296 I am not sure, if that's the right way to use it, you could request change in name of alias `AdminTester` to something else, but in future when you change the step file location, it will mean change at only one place, going by your suggestion will mean changes at many places in the test since 'as' was not used @nhungredweb are you looking into this?

Reply:
Thanks @puneet0191. But I think it should change  name of alias AdminTester  to something else Because this name unknown meaning

Reply:
Changing name is fine, that's up to you, but asking to remove alias is a different thing